README.md

```diff
@@ -539,6 +543,9 @@ Fields listed under `task_definition` correspond to fields that will be included
   * `secrets` allows you to specify secrets which will be retrieved from SSM Parameter Store. See the [ECS Docs](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/specifying-sensitive-data.html) for more information, including how reference AWS Secrets Managers secrets from SSM Parameter Store.
     * `value_from` is the SSM Parameter ARN or name (if the parameter is in the same region as your ECS Task).
     * `name` is the name of the environment variable in which the secret will be stored.
+  * If you need to inject secrets into your logging configuration, you may set `secret_options` under `logging`. For more information, See the [logging secrets section](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/specifying-sensitive-data.html#secrets-logconfig) of the ECS docs.
+    * `value_from` is the SSM Parameter ARN or name (if the parameter is in the same region as your ECS Task).
```

Comment:
Secrets manager ARN is also supported

Reply:
Indeed! 😄 
https://github.com/aws/amazon-ecs-cli/pull/777/commits/f30c51f5c6fc8d587ab60ebb4b1b8245f414968e